### ISSUE TITLE
Issue on Zeit now.sh

### ISSUE THREAD

**Issue by robikovacs** [nuxt-community/svg-sprite-module#147]
Call stack:
```
FATAL  Cannot find module 'chokidar'
Require stack:
- /var/task/node_modules/@nuxtjs/svg-sprite/lib/module.js
- /var/task/node_modules/@nuxt/core/dist/core.js
- /var/task/now__launcher.js
- /var/runtime/UserFunction.js
- /var/runtime/index.js
  Error: Cannot find module 'chokidar'
  Require stack:
  - node_modules/@nuxtjs/svg-sprite/lib/module.js
  - node_modules/@nuxt/core/dist/core.js
  - now__launcher.js
  - /var/runtime/UserFunction.js
  - /var/runtime/index.js
  at Object.<anonymous> (node_modules/@nuxtjs/svg-sprite/lib/module.js:1)
2020-03-08T16:41:14.031Z	undefined	ERROR	λ Error while initializing nuxt: /var/task/node_modules/@nuxtjs/svg-sprite/lib/module.js:1
```

**Comment by robikovacs** [comment]
The web app works fine after this fix. 

Source: https://github.com/nuxt/now-builder

**Comment by farnabaz** [comment]
Fixed in [v0.4.10](https://github.com/nuxt-community/svg-sprite-module/releases/tag/v0.4.10)
